setup.py

```diff
@@ -33,7 +33,7 @@
 
 setup(
     name='ambari-presto',
-    version='1.1',
+    version='1.2',
```

Comment:
Oh sorry, I was following [this commit](https://github.com/prestodb/ambari-presto-service/commit/b69c2e5b225107aa8aee19b43e09bca880af9147) from @petroav